disclosures/2021-05-13.md

```diff
@@ -0,0 +1,69 @@
+# Incident disclosure 2021-05-13
+
+## Summary
+
+- A missing check in low-level call in `StrategyProxy` has lead to non-atomic execution of withdrawal transaction.
+- This resulted in transactions burning the shares but not performing the actual withdrawal when supplied with insufficient gas.
+- No funds were lost. Four users of crvAAVE vault have suffered from this issue. This has been remediated and the users have been made whole.
+- A new `StrategyProxy` has been deployed.
+
+## Background
+
+All Curve strategies indirectly interact with Yearn's `CurveYCRVVoter`[[1]](#References) contract to perform operations like `deposit` and `withdraw` against Curve gauges. For safety, these interactions are proxied through an access controlled `StrategyProxy`[[2]](#References) contract.
+
+A partially failed withdrawal attempt has revealed a flaw in the logic of `StrategyProxy` which may prevent underlying funds from being returned to the user in some scenarios.
+
+## Details of Incident
+
+Calls between `StrategyProxy` and `CurveYCRVVoter` use a low-level call function[[3]](#References) that returns either `true` or `false` to indicate if the call was successful.
+
+While `StrategyProxy` checks that the call was successful during `deposit` function calls (see 2020-09-25 disclosure[[4]](#References)), it was discovered that this same check is not performed during `withdraw`[[5]](#References) function calls. This leads to an undesirable scenario when the transaction can succeed despite a failure in the branched call to `CurveYCRVVoter`. The result is user's vault shares are burned with zero funds withdrawn from vault to user.
+
+The branched call has failed due to an "*out of gas*" error (see example transaction[[6]](#References)). 
+
+This error may have been caused by the higher amount of gas needed to claim the new `stkAAVE` token rewards from the two Curve gauges which have recently added them: `crvAAVE` and `sAAVE`. We have not seen this same "*out of gas*" failure on other gauges.
```

Comment:
the two vaults are `crvAAVE` and `crvSAAVE` (if we want to match what we display on the vaults page)

Reply:
For the sentence above and below